lib/fizzy/parser_expr.cpp

```diff
@@ -216,7 +251,8 @@ parser_result<Code> parse_expr(const uint8_t* pos, const uint8_t* end, uint32_t
         std::tie(opcode, pos) = parse_byte(pos, end);
 
         auto& frame = control_stack.top();
-        const auto& metrics = metrics_table[opcode];
+        const auto metrics = metrics_table[opcode];
+        const auto types = type_table[opcode];
```

Comment:
```suggestion
        const auto type = type_table[opcode];
```

Reply:
Ok, but I'll have to rename 4 variables for the block types in the switch below.